src/xebia_email_signature/templates/new_form.html

```diff
@@ -0,0 +1,517 @@
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/choices.js@10.2.0/public/assets/styles/choices.min.css"
+    />
+    <link href="/static/styles.css" rel="stylesheet" />
+  </head>
+  <body>
+    <main class="main">
+      <img class="logo" src="/static/xebia-logo.svg" alt="Xebia logo" />
+
+      <h1 class="site-title">Email Signature Generator</h1>
+
+      <div class="form-container">
+        <form
+          class="js-form"
+          action="/new/signature"
+          method="post"
+          enctype="multipart/form-data"
+          target="preview-iframe"
+          onsubmit="handleFormSubmit()"
+        >
+          <div class="form-group">
+            <label for="full_name">Name</label>
+            <input
+              type="text"
+              id="full_name"
+              name="full_name"
+              value="John Doe"
+              onblur="prefillEmail(this.value)"
+            />
+            <div id="full_name-error" class="error"></div>
+          </div>
+          <div class="form-group">
+            <label for="email">Email</label>
+            <input
+              type="text"
+              id="email"
+              name="email"
+              value="john.doe@xebia.com"
+              onchange="validateEmail()"
+            />
+            <div id="email-error" class="error"></div>
+          </div>
+
+          <div class="form-group">
+            <label for="job_role">Role</label>
+            <input
+              type="text"
+              id="job_role"
+              name="job_role"
+              value="Senior Designer | Netherlands"
+            />
+            <div id="job_role-error" class="error"></div>
+          </div>
+
+          <div class="form-group">
+            <label for="unit"
+              >Unit
+              <span
+                >Optional: If you leave this blank, this field is not going to
+                appear</span
+              ></label
+            >
+            <input
+              type="text"
+              id="unit"
+              name="unit"
+              value="Software Technology"
+            />
+            <div id="unit-error" class="error"></div>
+          </div>
+
+          <div class="form-group">
+            <label for="phone">Phone</label>
+            <input
+              placeholder="+14043434832"
+              type="text"
+              id="phone"
+              name="phone"
+              value="+00000000000"
+              onchange="validatePhoneNumber();"
+            />
+            <div id="phone-error" class="error"></div>
+          </div>
+
+          <div>
+            <div class="js-form-sm" data-clone-count="0">
+              <div class="form-section">
+                <div class="form-group select-icon-picker">
+                  <label for="sm-0-icon">Icon</label>
+                  <select class="js-choice" id="sm-0-icon" name="sm[0][icon]">
```

Comment:
I Noticed that the Github option is gone. For a lot of techies this is going to be the wrong message.

Reply:
It's by design. You have to discuss about it with Tomek Zmarzły.